qt-models/diveplannermodel.h

```diff
@@ -73,7 +75,6 @@ public
 	void setVpmbConservatism(int level);
 	void setSurfacePressure(int pressure);
 	void setSalinity(int salinity);
-	int getSurfacePressure();
```

Comment:
I'm don't think this is the case here (as git blame points to early commits from Tomaz) but one way to make things accessible from QML is to make it a slot (`Q_INVOKABLE` is the cleaner one - but at least early on I didn't realize that so I used slots for quite a few of these).